components/src/form-field/suffix.directive.ts

```diff
@@ -0,0 +1,18 @@
+import { Directive, HostBinding, Input } from "@angular/core";
+
+import { PrefixClasses } from "./prefix.directive";
+
+@Directive({
+  selector: "[bitSuffix]",
```

Comment:
What do you think about making these selectors more specific, like `button[bitSuffix]`? That way they indicate their intended usage (i.e. "use this with a button element")

Reply:
We should probably also support non-buttons for text values.